apps/condo/domains/acquiring/constants/recurrentPayment.js

```diff
@@ -0,0 +1,23 @@
+// list of recurrent payment statuses
+const RECURRENT_PAYMENT_INIT_STATUS = 'CREATED'
+const RECURRENT_PAYMENT_PROCESSING_STATUS = 'PROCESSING'
+const RECURRENT_PAYMENT_DONE_STATUS = 'DONE'
+const RECURRENT_PAYMENT_ERROR_NEED_RETRY_STATUS = 'ERROR_NEED_RETRY'
+const RECURRENT_PAYMENT_ERROR_STATUS = 'ERROR'
+
+const RECURRENT_PAYMENT_STATUSES = [
+    RECURRENT_PAYMENT_INIT_STATUS,
```

Comment:
Should it have transitions, frozen fields, required fields same as payment / multipayment?

Reply:
Nope. Transitions of statuses going to be done on the server side. No frozen/required fields constants required